conftest.py

```diff
@@ -49,6 +49,14 @@ def pytest_collection_modifyitems(config, items):
                 )
             )
             item.add_marker(marker)
+        # OpenML is not working for PyPy
+        elif (item.name.endswith("test_openml") and
+                platform.python_implementation() == "PyPy"):
+            marker = pytest.mark.xfail(
+                reason="Known failure of OpenML for PyPy. See: "
+                       "https://github.com/scikit-learn/scikit-"
+                       "learn/issues/18906 for details.")
+            item.add_marker(marker)
```

Comment:
It would be better to use `@fails_if_pypy` decorator on the appropriate test with this comment.

Reply:
I did not know that 😉